src/components/SwapWidget/LimitOrder/index.tsx

```diff
@@ -163,6 +173,19 @@ const LimitOrder: React.FC<Props> = ({ swapType, setSwapType, isLimitOrderVisibl
     [onUserInput],
   );
 
+  // setting default tokens
+  const defaultInputCurrency = unwrappedToken(useToken_(defaultInputAddress) as Token, chainId);
+  const defaultOutputCurrency = unwrappedToken(useToken_(defaultOutputAddress) as Token, chainId);
```

Comment:
changes looks good thank you. 

1. But there are some minor changes needs to be handled. To make it more readable always use hook as a separate expression instead of composing with other expressions. for e.g.
```ts
const defaultInputToken = useToken(defaultInputAddress)
const defaultInputCurrency = defaultInputToken ? unwrappedToken(defaultInputToken, chainId) : undefined;
```

2. no need to append `_` after `useToken` in this case.
3. make useEffect observe on `defaultInputCurrency` & `defaultOutputCurrency` because thats what you are using in useEffect